.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[alias]
+xtask = "run --package xtask --"
```

Comment:
I thought you leaned toward multiple binaries (https://github.com/matklad/cargo-xtask/issues/2)

Reply:
I wonder if we could build each binary on demand. `xtask` binary acts as a thin layer of parent command delegates other subcommands.

That is, when running `cargo xtask unpublished` it spawns a process running `cargo run -p xtask --bin unpublished`. With that we can probably avoid polling too many dependencies.

Reply:
I am thinking of making `src/doc/build.sh` an xtask, which shouldn't compile the whole Cargo when running.

Reply:
I was wrong. That mean we need `unpublished` to be a separate bin. However, I feel like we should go multi-binaries way to make build time less suck.

Reply:
I've gone ahead and created a dedicated binary